.gitignore

```diff
@@ -4,3 +4,8 @@ dist/
 .temp/
 .DS_Store
 .env*
+
+# Generated client documentation (keep _template.md and clients.json)
+docs/clients/*.md
```

Comment:
If this already lives in the main README is there much benefit to including these?